game/ato/traveltime.py

```diff
@@ -18,6 +18,9 @@ def for_flight(cls, flight: Flight, altitude: Distance) -> Speed:
         # on fuel, but mission speed will be fast enough to keep the flight
         # safer.
 
+        if flight.squadron.aircraft.cruise_speed is not None:
+            return flight.squadron.aircraft.cruise_speed
```

Comment:
Hmm. Specifying this in knots works well for low flying aircraft, but this is potentially problematic for aircraft that cruise at higher altitudes. The speeds we give to DCS are in ground speed iirc, not IAS. 350 knots ground speed at sea level is a pretty reasonable cruise speed for an F-16 at sea level, but it's very slow at 30k ft.

Two options that come to mind (and they're not mutually exclusive):

1. Also allow `cruise_speed_mach` in the yaml (emit an error if both are set)
2. Make the yaml `cruise_speed_kt_indicated` and use the same trick on L37-L37 to approximate that? It won't be completely correct because we don't currently account for wind in that, but we probably could start doing that.

Thoughts?

Reply:
Makes sense, indicated airspeed would be more intuitive as the configured value would line up with what was indicated in the cockpit. 

I went with option 2, knots is a bit more intuitive than a Mach number for me. However, as you say, it doesn't prevent us from introducing a `cruise_mach` parameter later if there's a use case.